### ISSUE TITLE
Call a function running locally

### ISSUE THREAD

**Issue by joserocha3** [n6g7/redux-saga-firebase#97]
Is there a way to call a [function running locally](https://firebase.google.com/docs/functions/local-emulator)?

Instead of `https://us-central1-project-id.firebaseapp.com/` it would run at say `http://localhost:5000/`.
```javascript
function* callFunction() {
  // Will make a POST request to https://us-central1-project-id.firebaseapp.com/sayHello?name=Alfred
  // with custom headers
```

**Comment by n6g7** [comment]
Hi Jose,

That's a good idea, I hadn't thought about this! 🙂

I can see two options to do that:
- allow the first argument of `functions.call` to be either a function name (current behaviour) or a full URL. In the latter case we wouldn't automatically generate a xxx.cloudfunctions.net URL and just use what was passed by the user.
  ```js
  // Calls 'http://localhost:5000/greet?name=Jose':
  yield rsf.functions.call(
    'http://localhost:5000/greet',
    { name: 'Jose' }
  )
  ```
- create a new `baseUrl` property in `rsf` that is used to generate the url, ie:
  ```js
  rsf.baseUrl = 'http://localhost:5000/'

  [...]

  // Calls 'http://localhost:5000/greet?name=Jose':
  yield rsf.functions.call(
    'greet',
    { name: 'Jose' }
  )
  ```
  If not specified, we'd default to a xxx.cloudfunctions.net url.

Thoughts?
I'm inclined to go with the first option because it's more flexible, although it does require the user to provide (and build) the full URL for each call.

**Comment by joserocha3** [comment]
I like that first one. People might run on different local hosts so they
should have to enter the entire URL if running locally. BTW great library,
intuitive and fun to use -- appreciate all the work put into it.

**Comment by n6g7** [comment]
@joserocha3 Thanks for the encouragement!

This is now available in [v0.11](https://github.com/n6g7/redux-saga-firebase/releases/tag/v0.11.0), let me know how that works for you. :)
